R/estimate_infections.R

```diff
@@ -35,6 +35,11 @@
 #' @param horizon Numeric, defaults to 7. Number of days into the future to
 #' forecast.
 #'
+#' @param weigh_prior_delays Logical. If TRUE (default), all delay distribution
+#' priors will be weighted by the number of observation data points, usually
+#' preventing the posteriors from shifting. If FALSE, no weight will be applied,
```

Comment:
Could add: This approximates placing an independent prior on each time point but with less computational overhead.